api/libproject/generate_appengine_web_xml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+CIRCLECI="${CIRCLECI:-false}"
+
+if [[ ! "$CIRCLECI" == "true" ]]; then
+  cd db
+  source vars.env
+  cd ..
+else
+  WORKBENCH_DB_PASSWORD="$WORKBENCH_PASSWORD"
```

Comment:
Added. (I can remove the old one later.)

Please make sure to remove any other references to WORKBENCH_PASSWORD in the code.